### PR TITLE
updated docs

### PR DESCRIPTION
**Pull request by michal-dagan**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 1]
```diff
@@ -76,13 +76,14 @@ Alternatively, instead of providing the *Client Secret*, you can authenticate us
...
 
-### Authorize on Behalf of a User
+### Authorization Code flow
 Some Cortex XSOAR-Microsoft integrations (e.g., Microsoft Graph Mail Single User) require authorization on behalf of a user (not admin consent). For more information about this authorization flow, refer to the [authorization code flow](https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow).
 
 To configure a Microsoft integration that uses this authorization flow with a self-deployed Azure application:
 
 1. Make sure the needed permissions are granted for the app registration, e.g., for Microsoft Graph User: API/Permission name `Directory.AccessAsUser.All` of type `Delegated`.
-2. The Redirect URI can direct any web application that you wish to receive responses from Azure AD. If you are not sure what to set, you can use `https://localhost`. 
+2. In your app click **Authentication > Platform configurations > Add a platform.** Choose **Web** and add Redirect URI: https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-redirect-uri,
```
Comment (michal-dagan) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:85]:
```suggestion
2. In your app click **Authentication > Platform configurations > Add a platform.** Choose **Web** and add Redirect URI: https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-redirect-uri.
```

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 2]
```diff
@@ -76,13 +76,14 @@ Alternatively, instead of providing the *Client Secret*, you can authenticate us
...
 
-### Authorize on Behalf of a User
+### Authorization Code flow
 Some Cortex XSOAR-Microsoft integrations (e.g., Microsoft Graph Mail Single User) require authorization on behalf of a user (not admin consent). For more information about this authorization flow, refer to the [authorization code flow](https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow).
 
 To configure a Microsoft integration that uses this authorization flow with a self-deployed Azure application:
 
 1. Make sure the needed permissions are granted for the app registration, e.g., for Microsoft Graph User: API/Permission name `Directory.AccessAsUser.All` of type `Delegated`.
-2. The Redirect URI can direct any web application that you wish to receive responses from Azure AD. If you are not sure what to set, you can use `https://localhost`. 
+2. In your app click **Authentication > Platform configurations > Add a platform.** Choose **Web** and add Redirect URI: https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-redirect-uri.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:85]:
```suggestion
2. In your app. click **Authentication** > **Platform configurations** > **Add a platform.** Choose **Web** and add Redirect URI: https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-redirect-uri.
```

**Comment by ShirleyDenkberg** [comment]
@michal-dagan Doc review completed.

**Review by yucohen — APPROVED** [review]

File: content-repo/extra-docs/articles/microsoft-auth-guide.md [hunk 3]
```diff
@@ -76,13 +76,14 @@ Alternatively, instead of providing the *Client Secret*, you can authenticate us
...
 
-### Authorize on Behalf of a User
+### Authorization Code flow
 Some Cortex XSOAR-Microsoft integrations (e.g., Microsoft Graph Mail Single User) require authorization on behalf of a user (not admin consent). For more information about this authorization flow, refer to the [authorization code flow](https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow).
 
 To configure a Microsoft integration that uses this authorization flow with a self-deployed Azure application:
 
 1. Make sure the needed permissions are granted for the app registration, e.g., for Microsoft Graph User: API/Permission name `Directory.AccessAsUser.All` of type `Delegated`.
-2. The Redirect URI can direct any web application that you wish to receive responses from Azure AD. If you are not sure what to set, you can use `https://localhost`. 
+2. In your app. click **Authentication** > **Platform configurations** > **Add a platform.** Choose **Web** and add Redirect URI: https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-redirect-uri.
```
Comment (michal-dagan) [inline content-repo/extra-docs/articles/microsoft-auth-guide.md:85]:
```suggestion
2. In your app. click **Authentication** > **Platform configurations** > **Add a platform.** Choose **Web** and add [Redirect URI](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-redirect-uri).
```

**Pull request merged**